frontend/src/utils/focus-management.ts

```diff
@@ -88,8 +88,11 @@ export function isFocusableElement(element: HTMLElement) {
   return element.matches(focusableSelector)
 }
 
-export function focusElement(element: HTMLElement | null) {
-  element?.focus()
+export function focusElement(element: HTMLElement | Element | null) {
+  if (!element || !(element instanceof HTMLElement)) {
+    return
+  }
+  element.focus()
```

Comment:
`!(!c1 || !c2)` can be simplified to `c1 && c2`. Without the double negative, the condition is more readable. It can even be simplified further because `null` and `undefined` aren't instances of `HTMLElement`.

```suggestion
  if (element instanceof HTMLElement) {
    element.focus()
  }
```